### DISCUSSION TITLE
Can I use a custom clang-tidy binary instead of the built-in one?

### DISCUSSION THREAD

**Discussion by MoonChasing** [clangd/clangd#1646] [Q&A] [answered]
Can I use a custom clang-tidy binary instead of the built-in one?

**Comment by HighCommander4** [comment, marked as answer]
No, this is not supported.

Clangd does not invoke an external clang-tidy binary, it links to the clang-tidy libraries and invokes their functionality inside the clangd process.

**Reply by HighCommander4** [reply]
See also some relevant discussion at https://github.com/clangd/clangd/issues/1458

**Reply by MoonChasing** [reply]
Thanks.
